sdk/go/common/workspace/plugins.go

```diff
@@ -380,7 +386,9 @@ func (info PluginInfo) Install(tarball io.ReadCloser) error {
 		// TODO[pulumi/pulumi#1334]: move to the language plugins so we don't have to hard code here.
 		switch runtime {
 		case "nodejs":
-			if _, err := npm.Install(finalDir, nil, os.Stderr); err != nil {
+			var b bytes.Buffer
+			if _, err := npm.Install(finalDir, &b, &b); err != nil {
+				os.Stderr.Write(b.Bytes())
```

Comment:
Any chance we can easily fix https://github.com/pulumi/pulumi/issues/5747 while we are here?

Reply:
I could make some additional targeted changes here to work towards fixing #5747, but I think there are some other places that would need to be fixed (not showing the progress bar, other places we output "installing" messages, etc.), so would rather hold off and tackle that altogether to ensure it's truly addressed.